frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -63,23 +68,23 @@ export default class AccountsApi extends ExplorerApi {
 
   async getAccounts(
     limit: number = 15,
-    endTimestamp?: number
+    paginationIndexer?: AccountPagination
   ): Promise<AccountBasicInfo[]> {
     try {
       return await this.call("select", [
         `SELECT account_id as id, created_at_block_timestamp as createdAtBlockTimestamp, 
           created_by_transaction_hash as createdByTransactionHash, account_index as accountIndex
           FROM accounts
           ${
-            endTimestamp
-              ? `WHERE (created_at_block_timestamp * 1000000 + account_index) < :endTimestamp`
+            paginationIndexer
+              ? `WHERE created_at_block_timestamp < :endTimestamp OR (created_at_block_timestamp = :endTimestamp AND account_index <= :accountIndex)`
               : ""
           }
-          ORDER BY (created_at_block_timestamp * 1000000 + account_index) DESC
+          ORDER BY created_at_block_timestamp DESC, account_index DESC
```

Comment:
This query does not make sense to me.

Let's consider the following example:

```
account_id                     | created_at_block_timestamp
lol.test                       | 1
anon.test                      | 1
marcink60.test                 | 1
ff.test                        | 3
damons.test                    | 3
vitalik.test                   | 5
dev-1587569727975              | 6
marketm.test                   | 7
studio5215580270               | 7
bitcoinwallah.test             | 7
chironcoin_platform.test       | 10
studio-wn9p1yo8t               | 11
studio-a2qvmp7xk               | 12
studio-m4gn1ukx5               | 13
chironcoin_celine.test         | 16
chironcoin_matthieu.test       | 16
chironcoin_abhi.test           | 16
pengan.test                    | 16
dev-1587540578053              | 18
test-pr-517-ledger.test        | 19
```

```sql
SELECT
ORDER BY created_at_block_timestamp DESC
LIMIT 4
```

Output:

```
test-pr-517-ledger.test        | 19
dev-1587540578053              | 18
pengan.test                    | 16
chironcoin_abhi.test           | 16
```

after that auto-refresh does:

```sql
SELECT WHERE created_at_block_timestamp < 16
ORDER BY created_at_block_timestamp DESC
LIMIT 4
```

```
studio-m4gn1ukx5               | 13
studio-a2qvmp7xk               | 12
studio-wn9p1yo8t               | 11
chironcoin_platform.test       | 10
```

and after another refresh, it should go even further...

Reply:
I tested it manually with real database and some dumb added account, the query is not like what you show, it will query all 16 rows since account_index is different, and then query 13, 12,11,10

Reply:
Please, set the page size to 4 and list all the queries and the output we get from Explorer frontend.